### ISSUE TITLE
trtAssign return integer or factor

### ISSUE THREAD

**Issue by elinsooon** [kgoldfeld/simstudy#151]
fix trtAssign to return type factor (or integer)

**Comment by elinsooon** [comment]
is it sufficient for trtAssign to just return type integers? Once it's in this form it can be easily coerced to type factor, correct? If it is sufficient, we can close this issue

**Comment by kgoldfeld** [comment]
I don't think it would be a great idea to force it to be a factor at this point, because it might mess with some existing user code, but we might want to consider adding an explicit argument to create a factor (e.g. `asFactor`). So, maybe we can keep it open for now in case we decide to do that.
